python/usb_console/radio_session.py

```diff
@@ -10,6 +10,7 @@
 import base64
 import os
 import email
```

Comment:
We can probably pare down a good number of these import statements, now that a lot of radio session logic got moved to ground software